compiler/src/dmd/expression.d

```diff
@@ -249,18 +249,54 @@ bool isDotOpDispatch(Expression e)
 }
 
 /****************************************
- * Expand tuples.
- * Input:
- *      exps    aray of Expressions
- * Output:
- *      exps    rewritten in place
+ * Expand tuples in-place.
```

Comment:
I think this function needs more proper documentation to explain what the `names` parameter represents. I, personally, find that just describing the parameter is not sufficient for someone new to the codebase to understand.

Reply:
I added an example to clarify